### ISSUE TITLE
Failed to register GATT advertisement: org.freedesktop.DBus.Error.NoReply

### ISSUE THREAD

**Issue by robputt** [NebraLtd/hm-config#42]
```
28.06.21 22:14:15 (+0000)  gateway-config  DEBUG:root:Wi-Fi Complete
28.06.21 22:14:30 (+0000)  gateway-config  DEBUG:root:Wi-Fi Scanning
28.06.21 22:14:30 (+0000)  gateway-config  DEBUG:root:Wi-Fi Complete
28.06.21 22:14:35 (+0000)  gateway-config  Failed to register GATT advertisement: org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
```

**Comment by MuratUrsavas** [comment]
The problem turned out to be the Bluetooth firmware, like we mentioned earlier. Radxa released a new firmware for it an we've shared it with Balena to release a new OS revision. I'll be tracking the progress.

Note: The firmware is working fine on an AW-NB197SM but yet to see the effects on an AP6212.

**Comment by MuratUrsavas** [comment]
Confirmed on testnet, everything is fine right now. All has been fixed with the 2022.1.4 RockPi BalenaOS release. (Included in 2022.04.15+ production images)
